### PR TITLE
Label section added, FCC frequency bands updated

### PR DESCRIPTION
**Pull request by jcarolinares**
## What This PR Changes
- Label section added, FCC frequency bands updated

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md [hunk 1]
```diff
@@ -407,9 +413,12 @@ Lors de l’ installation et de l’ exploitation de ce dispositif, la distance
...
 Hereby, Arduino S.r.l. declares that this product is in compliance with essential requirements and other relevant provisions of Directive 201453/EU. This product is allowed to be used in all EU member states.
 
-| Frequency bands      | Typical Output Power |
-|----------------------|----------------------|
-| 2.4 GHz, 40 channels | TBC                  |
+| Frequency bands                 | Maximum Output Power |
+| ------------------------------- | -------------------- |
+| 2402 MHz ~ 2480 MHz (EDR)       | -0.21 dBM            |
+| 2402 MHz ~ 2480 MHz (BLE)       | 4.79 dBM             |
+| 2412 MHz ~ 2462 MHz (2.4G WiFi) | 16.21 dBM            |
```
Comment (mcmchris) [inline content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md:420]:
```suggestion
| 2412 MHz ~ 2462 MHz (2.4GHz WiFi) | 16.21 dBm            |
```

File: content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md [hunk 2]
```diff
@@ -407,9 +413,12 @@ Lors de l’ installation et de l’ exploitation de ce dispositif, la distance
 
 Hereby, Arduino S.r.l. declares that this product is in compliance with essential requirements and other relevant provisions of Directive 201453/EU. This product is allowed to be used in all EU member states.
 
-| Frequency bands      | Typical Output Power |
-|----------------------|----------------------|
-| 2.4 GHz, 40 channels | TBC                  |
+| Frequency bands                 | Maximum Output Power |
+| ------------------------------- | -------------------- |
+| 2402 MHz ~ 2480 MHz (EDR)       | -0.21 dBM            |
+| 2402 MHz ~ 2480 MHz (BLE)       | 4.79 dBM             |
```
Comment (mcmchris) [inline content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md:419]:
```suggestion
| 2402 MHz ~ 2480 MHz (BLE)       | 4.79 dBm             |
```

File: content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md [hunk 3]
```diff
@@ -407,9 +413,12 @@ Lors de l’ installation et de l’ exploitation de ce dispositif, la distance
 
 Hereby, Arduino S.r.l. declares that this product is in compliance with essential requirements and other relevant provisions of Directive 201453/EU. This product is allowed to be used in all EU member states.
 
-| Frequency bands      | Typical Output Power |
-|----------------------|----------------------|
-| 2.4 GHz, 40 channels | TBC                  |
+| Frequency bands                 | Maximum Output Power |
+| ------------------------------- | -------------------- |
+| 2402 MHz ~ 2480 MHz (EDR)       | -0.21 dBM            |
```
Comment (mcmchris) [inline content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md:418]:
```suggestion
| 2402 MHz ~ 2480 MHz (EDR)       | -0.21 dBm            |
```

**Review by mcmchris — CHANGES_REQUESTED** [review]
@jcarolinares , I suggested some unit changes in writing. Everything else, LGTM

File: content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md (hunk 1 above)
Comment (jcarolinares) [reply content/hardware/06.nicla/boards/nicla-vision/datasheet/datasheet.md:420]: Based on https://en.wikipedia.org/wiki/DBm this is correct, added comment about the arduino style guide too

**Review by mcmchris — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**